pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -140,6 +144,9 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 	if packetOutType == Unsupported {
 		return fmt.Errorf("error when generating reject response for the packet from: %s to %s: neither source nor destination are on this Node", dstIP, srcIP)
 	}
+	if packetOutType == RejectServiceRemoteToExternal {
+		dstMAC = "aa:bb:cc:dd:ee:ff"
```

Comment:
why is this the mac represent tunnel instead of gateway? I think it wants to send via gateway?

Reply:
It simulates reject packets sent by a remote Node toward the current Node. These packets use tun MAC as theirs dst MAC.

Reply:
got it. but we should use `GlobalVirtualMAC` here?

Reply:
Oh yes. Done.